src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -732,7 +744,12 @@ function build_eal_list() {
 		'*Remote Gateway',
 		'text',
 		$pconfig['remotegw']
-	))->setHelp('Enter the public IP address or host name of the remote gateway.');
+	))->setHelp('Enter the public IP address or host name of the remote gateway.%1$s%2$s%3$s',
+	    '<div class="infoblock">',
+	    sprint_info_box(gettext('Use \'0.0.0.0\' to allow connection from any IPv4 address or \'::\' ' . 
+	    'to allow connection from any IPv6 address.' . '<br/>' . 'It supports only Responder mode, ' . 
+	    'can\'t use the peer IP address for the remote identifier, and limited to a single entry.'), 'info', false),
```

Comment:
Starting with `It supports only Responder mode,` this needs reworded as it's a bit awkward/incorrect grammar.

Perhaps: "Responder Only must be set, Peer IP Address cannot be used for Remote Identifier, and only a single Phase 1 entry may use this Remote Gateway value."

Reply:
ok, all fixed
but you can use 0.0.0.0 for multiple PH1 (tested, see above)

Reply:
If it works OK, that can be removed then.

Reply:
'and limited to a single entry.' is already removed